blueprints/app/files/package.json

```diff
@@ -15,7 +15,7 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "<% if (yarn) { %>yarn lint:hbs<% } else { %>npm run lint:hbs<% } %> && ember test"
```

Comment:
We could also utilize "pretest" as well. I'm not sure of the advantages of each. This is essentially equivalent:

```
"pretest": "npm run lint:js",
"test": "ember test"
```

Reply:
Compatibility with Windows is the advantage of `pretest`.